src/OmniSharp.LanguageServerProtocol/Handlers/CompletionHandler.cs

```diff
@@ -80,22 +80,28 @@ public async Task<CompletionList> Handle(CompletionParams request, CancellationT
                 Line = Convert.ToInt32(request.Position.Line),
                 WantKind = true,
                 WantDocumentationForEveryCompletionResult = true,
-                WantReturnType = true
+                WantReturnType = true,
+                WantSnippet =_capability.CompletionItem.SnippetSupport,
```

Comment:
We should probably check for null.

Change to `_capability.CompletionItem?.SnippetSupport ?? false`?